src/main/java/core/basesyntax/interfaces/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Drawable {
+    String drawInfo();
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/3296ba47-7ce1-4413-9c99-b65e59c9bf97)
```suggestion
    void drawInfo();
```